catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/faceting/FacetValueCountImpl.java

```diff
@@ -14,7 +14,7 @@
 package ddf.catalog.operation.faceting;
 
 /** A simple tuple object pairing text faceting value results with their respective counts. */
-public class FacetValueCount {
+public class FacetValueCountImpl implements FacetValueCount {
```

Comment:
Missing `@Override` annotations on `getCount()` and `getValue()`

Reply:
done